### PR TITLE
Fix Draco accessor ids

### LINKED ISSUE #6: GLTFScene throws an exception trying to render motional-car01.glb
This can be reproed on https://webviz.io/worldview/#/docs/commands/gltfscene by using https://jhurliman.s3.amazonaws.com/motional-car01.glb as the model parameter. That model renders fine with https://gltf-viewer.donmccurdy.com/ though.

```
Uncaught Error: (regl) invalid dynamic attribute "position" in command https://webviz.io/dist/docs.js:2:263080
    at raise (regl.js:47)
    at Function.commandRaise (regl.js:390)
    at batch (eval at compile (regl.js:5549), <anonymous>:1370:223)
    at regl.js:9432
    at GLTFScene.js:247
    at scope (eval at compile (regl.js:5549), <anonymous>:382:1)
    at regl.js:9424
    at GLTFScene.js:246
    at WorldviewContext.js:432
    at Array.forEach (<anonymous>)
```

### PR DESCRIPTION
**Pull request by jtbandes**
- Update Webpack config to make `yarn storybook` + `yarn watch` work. `~` pseudo-package name is now used for self-imports.
- Move `.glb` files to Git LFS.
- Fix #6 — the Draco decompression functions were incorrectly using accessor ids from the original `attributes` rather than the ones loaded from the compressed geometry. The sample model motional-car01.glb is from https://www.nureality.org

I'm still not sure we implement this part of the spec correctly:

> If additional attributes are defined in `primitive`'s `attributes`, but not defined in `KHR_draco_mesh_compression`'s `attributes`, then the loader must process the additional attributes as usual.

But at least things seem to be working for our sample data so far.

image

### CODE REVIEW

File: src/utils/draco.js [hunk 1]
```diff
@@ -83,20 +81,19 @@ const decodeIndices = (draco, decoder, dracoGeometry) => {
...
+  KHR_draco_mesh_compression.accessors = [];
 
-  const { attributes } = dracoCompression;
-  dracoCompression.accessors.push(...decodeAttributes(draco, decoder, dracoGeometry, attributes));
-
-  dracoCompression.accessors.push(decodeIndices(draco, decoder, dracoGeometry));
+  for (const attributeId of Object.values(KHR_draco_mesh_compression.attributes)) {
+    KHR_draco_mesh_compression.accessors[attributeId] = decodeAttribute(draco, decoder, dracoGeometry, attributeId);
+  }
+  KHR_draco_mesh_compression.accessors[primitive.indices] = decodeIndices(draco, decoder, dracoGeometry);
```
Comment (jtbandes) [inline src/utils/draco.js:96]: This is the main fix for motional-01.glb.

File: src/commands/GLTFScene.js [hunk 2]
```diff
@@ -168,23 +168,25 @@ const drawModel = (regl) => {
...
 
         let primitiveAccessors = accessors;
+        let primitiveAttributes = primitive.attributes;
         const { extensions = {} } = primitive;
         const dracoCompressionEXT = extensions.KHR_draco_mesh_compression;
         if (dracoCompressionEXT) {
           // If mesh contains compressed data, accessors will be available inside
           // the draco extension. See `parseGLB.js` and `draco.js` files.
           primitiveAccessors = dracoCompressionEXT.accessors;
+          primitiveAttributes = dracoCompressionEXT.attributes;
```
Comment (jtbandes) [inline src/commands/GLTFScene.js:178]: This is the main fix for motional-01.glb.

File: docs/src/2.4.ManagingTheCamera.mdx [hunk 3]
```diff
@@ -17,7 +17,7 @@ In this step, we'll add a duck model to the center of the scene, and move the ca
 First, let's add a duck to the scene using [GLTFScene](#/docs/commands/gltfscene), and add the `Duck.glb` model file to the `src/utils` folder.
 
 ```js
-import duckModel from "common/fixtures/Duck.glb"; // Webpack magic: we actually import a URL pointing to a .glb file
+import duckModel from "~/common/fixtures/Duck.glb"; // Webpack magic: we actually import a URL pointing to a .glb file
```
Comment (jhurliman) [inline docs/src/2.4.ManagingTheCamera.mdx:20]: I assume this means something other than "import from your $HOME directory"

Comment (jtbandes) [reply docs/src/2.4.ManagingTheCamera.mdx:20]: yeah, it's configured in webpack resolve.alias. I stole this idea from @defunctzombie

Comment (jtbandes) [reply docs/src/2.4.ManagingTheCamera.mdx:20]: The previous state of affairs was that the whole project root directory was being used as a path to resolve modules, which I didn't like

**Pull request merged**